packages/base-styles/test/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+const util = require( 'util' );
+const nodeSass = require( 'node-sass' );
+const path = require( 'path' );
+
+const run = ( filename ) => {
+	return util.promisify( nodeSass.render )( {
+		file: path.resolve( __dirname, filename ),
+	} );
+};
+
+describe( 'base-styles', () => {
+	it( 'can use animations', () => {
+		return run( 'animations.scss' ).then( ( result ) => {
+			expect( result.css.toString() ).toMatchSnapshot();
```

Comment:
What's the reasoning to test CSS snapshots? these can change way too often IMO.

Reply:
Good question!

The main thing is to ensure we don't output any CSS from these files unless intended.

Another (little less important IMHO) is that since these files are pretty highly dependant on each other _in global scope_, modifying one can have unintended consequences to another. Not sure if that's worth testing for.

@youknowriad @gziolo Is it better to test a snapshot of importing them all and ensuring that just comments are coming out?

Reply:
I'm personally thinking about another developper coming later and seeing a breakage here. He probably would have changed the content of that CSS so the reflex will automatically be to update the snapshots without further thinking.

I don't have good suggestions here but IMO sometimes no testing is better than testing where the expectations are not clearly defined in the test.

Reply:
Makes sense 👍 Removed tests in https://github.com/WordPress/gutenberg/pull/17883/commits/68dd6067722a8fc999eb452507a93127dd11b02d